src/factories/Section.php

```diff
@@ -97,8 +99,18 @@ function definition(int $index = 0) {
      * @param \craft\models\Section $element
      */
     function store($element) {
-        \Craft::$app->sections->saveSection($element);
+
+        if (\Craft::$app->sections->saveSection($element) === false) {
+          throw new InvalidArgumentException(
+              "Unable to store new section: " .
+              print_r($element->toArray(), true) .
+              print_r($element->getErrors(), true)
+          );
```

Comment:
This happens in so many places, I should make an `ModelSaveException` that just takes an `$model` in the constructor and handles this for us. I have almost the exact same code in the `factories\Entry::store()` method.